.gitattributes

```diff
@@ -1 +1,3 @@
 src/catalog_manager/_version.py export-subst
+# SCM syntax highlighting & preventing 3-way merges
+pixi.lock merge=binary linguist-language=YAML linguist-generated=true
```

Comment:
This is generated by `pixi init` not sure what that means. Will have to look later...